lib/lifecycles/changelog.js

```diff
@@ -32,11 +39,16 @@ function outputChangelog (args, newVersion) {
     }
     let content = ''
     const context = { version: newVersion }
+    const wirterOptions = {
```

Comment:
Should this be "writerOptions"?

Reply:
you are absolutely right.

Reply:
👍 really looking forward to this being merged. Is there anything I can do to help?